server/src/people-service.js

```diff
@@ -6,7 +6,16 @@ import {sendEmail} from './util/email';
 import {NotFoundError, RequestError} from './util/error-util';
 import type {PersonType} from './types';
 
-const config = {database: 'demo'};
+const config = {
+    user: process.env.SQL_USER,
+    password: process.env.SQL_PASSWORD,
+    database: process.env.SQL_DATABASE
```

Comment:
How do you feel about adding "|| 'demo' to the end of line 12 so it works locally when the SQL_DATABASE environment variable is not set?

Reply:
Done.